.studyminimeta.yaml

```
@@ -0,0 +1,155 @@
+    and brain structure, to sensory integration and attentional processes, to computational modeling of representational
+    spaces and brain area interactions."
+
+  # optional list of employed standard and format
+  # for automatic generation of rich metadata records
+  # (see handbook for full list)
+
+  # optional list of keyword/phrase
+  keyword:
+    - human
+    - fMRI
+    - task
+    - 7T
+    - 3T
+    - audio
+    - visual
+    - music
+    - retinotopy
+    - angiography
+    - T1,T2
+
+  # minimum of one dataset author required
+  author:
+    - g@fz-juelich.de
+
+  # optional list of entities that funded the creation of
+  # the dataset. This might also be a single value, i.e.:
+  #`` funding: NIH´´.
+  funding:
+    - BMBF, 01GQ1411
+    - NSF, 1429999
+
+
+# one or more publications on or using the dataset, optional
+publication:
+  - title: "The effect of acquisition resolution on orientation decoding from V1: comparison of 3T and 7T"
+    author:
+      - a@fz-juelich.de
+      - b@fz-juelich.de
+      - c@fz-juelich.de
+      - d@fz-juelich.de
+      - e@fz-juelich.de
+      - f@fz-juelich.de
+      - g@fz-juelich.de
+    # when published
+    year: 2018
+    # all other properties are optional
+    # highly recommended to provide a DOI
+    doi: https://doi.org/10.1101/305417
+    # journal name or publication venue label
+    publication: bioRxiv
+
+  - title: "Spatial band-pass filtering aids decoding musical genres from auditory cortex 7T fMRI"
+    author:
+      - a@fz-juelich.de
+      - f@fz-juelich.de
+      - g@fz-juelich.de
+    # when published
+    year: 2018
+    # all other properties are optional
+    # highly recommended to provide a DOI
+    doi: https://doi.org/10.12688/f1000research.13689.2
+    # journal name or publication venue label
+    publication: F1000Research
+
+  - title: "The effect of acquisition resolution on orientation decoding from V1 BOLD fMRI at 7 Tesla"
+    author:
+      - a@fz-juelich.de
+      - c@fz-juelich.de
+      - b@fz-juelich.de
+      - f@fz-juelich.de
+      - g@fz-juelich.de
+    # when published
+    year: 2017
+    # all other properties are optional
+    # highly recommended to provide a DOI
+    doi: https://doi.org/10.1016/j.neuroimage.2016.12.040
+    # journal name or publication venue label
+    publication: NeuroImage
+  
+  - title: "Ultra high-field multi-resolution fMRI data for orientation decoding in visual cortex"
+    author:
+      - a@fz-juelich.de
+      - b@fz-juelich.de
+      - c@fz-juelich.de
+      - d@fz-juelich.de
+      - e@fz-juelich.de
+      - f@fz-juelich.de
+      - g@fz-juelich.de
+    # when published
+    year: 2017
+    # all other properties are optional
+    # highly recommended to provide a DOI
+    doi: https://doi.org/10.1016/j.dib.2017.05.014
+    # journal name or publication venue label
+    publication: Data in Brief
+
+# person information, one record for every email-key used above
+person:
+  a@fz-juelich.de:
```

Since it would be a great example to follow, I decided to ask a clarification question:  why did you side for the meaningless a, b, c, d which might generally result in a confusion and misattribution (mixing up one person ID over another) in favor over some 
- clearly not a valid email
- including the actual name

```suggestion
  person-ayan-sengupta:
```
or even just

```suggestion
  ayan-sengupta:
```
or alike?